stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/options/message/internal/MessageOptionsDialogFragment.kt

```diff
@@ -164,12 +165,9 @@ internal class MessageOptionsDialogFragment : FullScreenDialogFragment() {
     }
 
     private fun setupMessageView() {
-        viewHolder = MessageListItemViewHolderFactory()
+        viewHolder = viewHolderFactory
             .apply {
                 decoratorProvider = MessageOptionsDecoratorProvider(style.itemStyle, style.replyMessageStyle)
```

Comment:
Can we move this initialization to the place where we are overriding `listenerContainer` (newInstance)? It looks a little bit weird to override factory's different things in different places